index.html

```diff
@@ -0,0 +1,563 @@
+                                    <path fill-rule="evenodd" clip-rule="evenodd" d="M17.1901 15.4783L23.6458 21.934C23.8727 22.161 24.0001 22.4689 24 22.7899C23.9999 23.111 23.8723 23.4188 23.6452 23.6457C23.4181 23.8726 23.1102 24 22.7892 23.9999C22.4682 23.9998 22.1604 23.8722 21.9335 23.6451L15.4778 17.1894C13.5479 18.6841 11.1211 19.3875 8.69104 19.1565C6.26097 18.9255 4.01018 17.7774 2.39656 15.9458C0.782936 14.1142 -0.0723147 11.7366 0.00479537 9.29681C0.0819054 6.857 1.08558 4.5382 2.81165 2.81214C4.53771 1.08607 6.85651 0.0823937 9.29632 0.00528365C11.7361 -0.0718264 14.1137 0.783424 15.9453 2.39705C17.7769 4.01067 18.925 6.26146 19.156 8.69153C19.387 11.1216 18.6836 13.5484 17.1889 15.4783H17.1901ZM9.60045 16.7994C11.5099 16.7994 13.3412 16.0409 14.6914 14.6907C16.0416 13.3405 16.8001 11.5092 16.8001 9.59974C16.8001 7.69027 16.0416 5.859 14.6914 4.5088C13.3412 3.1586 11.5099 2.40007 9.60045 2.40007C7.69098 2.40007 5.85971 3.1586 4.50951 4.5088C3.15932 5.859 2.40078 7.69027 2.40078 9.59974C2.40078 11.5092 3.15932 13.3405 4.50951 14.6907C5.85971 16.0409 7.69098 16.7994 9.60045 16.7994Z" fill="#737373"/>
+                                    </svg>
+
+                            </li>
+
+                                    <li>
+                                        <img src="./Images/Shop.png" alt="">
+                                    </li>
+                            </div>
+                          
+                                    
+                                <div class="nav__sub">
+                                    <li>
+                                            <svg width="24" height="14" viewBox="0 0 24 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                                                        <path
+                                                                            d="M0.571472 0H23.4286V2.28571H0.571472V0ZM6.28576 5.71429H23.4286V8H6.28576V5.71429ZM13.4286 11.4286H23.4286V13.7143H13.4286V11.4286Z"
+                                                                            fill="#737373" />
+                                                                    </svg>
+                                            
+                                                                </li>                        
+                                </div>
+    
+        
+                           
+                        </div>
+                </ul>
+            
+
+        
+
+        <div class="page1__body">
+            <h1 class="page1__heading">
+                Experts are here solve
+                your business problem.
+            </h1>
+            <p class="page1__para">
+                We know how large objects will act, but things on a
+                small scale just do not act that way.
+            </p>
+            <div class="page1__btns-block">
```

Comment:
I suggest you create a button as a block because you will need to reuse it in the whole project, if you create a button as an element of page1, it will be very hard to reuse it. so create a btn as an independent block that can be reused.